build/yamls/antrea.yml

```diff
@@ -1376,6 +1376,17 @@ data:
     # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
     # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
     #kubeAPIServerOverride: ""
+
+    # Comma-separated list of cipher suites. If omitted, the default Go cipher suites will be used.
+    # https://golang.org/pkg/crypto/tls/#pkg-constants
+    # Note that Antrea is using an old version of k8s.io/component-base, so TLS1.3 cipher suites cannot be explicitly listed here.
```

Comment:
I personally feel no need to explain the long reasons in yaml, but just describe the facts. We can move detailed explanation to comments on code. Up to you to decide.

Reply:
I feel that everything is needed, except for the `k8s.io/component-base` mention and the github link

Reply:
Yes, I meant "Antrea is using an old version of k8s.io/component-base" and "by default Antrea uses the Go "crypto" libraries".

Reply:
Sounds good. I had suggested using "by default Antrea uses the Go "crypto" libraries" because I was thinking of user which may have their own FIPS compliant builds of Antrea by using a different crypto implementation. But in that case, it's not really our responsibility to document the behavior.

Reply:
Removed component-base and the link.

Reply:
Also "Go crypto libraries"